source/admin_manual/mailbox_formats/maildir.rst

```diff
@@ -189,7 +189,6 @@ Filename Examples
 |                                                                                                   | that folder's ``dovecot-keywords`` file.                 |
 +---------------------------------------------------------------------------------------------------+----------------------------------------------------------+
 
-.. _`IMAP INTERNALDATE, RFC 3501 [2.3.3]`: https://tools.ietf.org/html/rfc3501#section-2.3.3
 .. _`index files`: https://wiki.dovecot.org/IndexFiles
```

Comment:
This is already migrated to the current docs:
https://doc.dovecot.org/developer_manual/design/indexes/mail_index_api/